### PR TITLE
Updating 2019-03-01 and 2019-07-01 compute schemas

### PR DESCRIPTION
**Pull request by mabhard**
Gallery schemas were moved to Microsoft.Compute.Galleries.json from Microsoft.Compute.json

### CODE REVIEW

File: schemas/2015-01-01/deploymentTemplate.json [hunk 1]
```diff
@@ -1380,8 +1380,6 @@
                   { "$ref": "https://schema.management.azure.com/schemas/2018-10-01/Microsoft.Compute.Extensions.json#/resourceDefinitions/virtualMachines_extensions" },
                   { "$ref": "https://schema.management.azure.com/schemas/2018-10-01/Microsoft.Compute.Extensions.json#/resourceDefinitions/virtualMachineScaleSets_extensions" },
                   { "$ref": "https://schema.management.azure.com/schemas/2019-03-01/Microsoft.Compute.json#/resourceDefinitions/availabilitySets" },
-                  { "$ref": "https://schema.management.azure.com/schemas/2019-03-01/Microsoft.Compute.json#/resourceDefinitions/galleries_applications" },
```
Comment (hyonholee) [inline schemas/2015-01-01/deploymentTemplate.json]: why you remove this?

Comment (mabhard) [reply schemas/2015-01-01/deploymentTemplate.json]: Gallery schemas were moved to Microsoft.Compute.Galleries.json from Microsoft.Compute.json

Comment (mabhard) [reply schemas/2015-01-01/deploymentTemplate.json]: The --tag=2019-07 removed these references in compute.json

**Comment by anthony-c-martin** [comment]
@hyonholee please could you confirm you're happy with these changes on behalf of Compute. I'm happy to sign off for ARM.

**Comment by hyonholee** [comment]
LGTM.

**Pull request merged**